src/APIFunctions/User.js

```diff
@@ -1,4 +1,5 @@
 import axios from 'axios'
+// import {ApiResponse, UserApiResponse} from './ApiResponses'
 
 /**
```

Comment:
if you havent already, copy and paste this comma block to the file with the classes

Reply:
the class is no longer here so please remove the comment block